libs/designsystem/src/lib/components/calendar/calendar.component.scss

```diff
@@ -7,27 +7,37 @@ table.calendar-table {
   user-select: none;
   outline: none !important;
   border-spacing: 0;
```

Comment:
`border-spacing` declaration can be removed. This property applies only when `border-collapse` is `separate`.